app/src/main/java/org/wikipedia/activity/BaseActivity.kt

```diff
@@ -54,7 +63,11 @@ abstract class BaseActivity : AppCompatActivity() {
     private var startTouchY = 0f
     private var startTouchMillis = 0L
     private var touchSlopPx = 0
+    var callback: Callback? = null
 
+    val requestPermissionLauncher = registerForActivityResult(ActivityResultContracts.RequestPermission()) { isGranted: Boolean ->
```

Comment:
Do you think it would be simpler to use  `it` instead of assigning a `Boolean` variable for it?

Reply:
I feel its more readable.